build.gradle

```diff
@@ -33,12 +39,7 @@ allprojects {
         }
         project.dependencies {
             implementation 'androidx.core:core-ktx:1.9.0'
-        }
-    }
-
-    tasks.withType(KotlinCompile).configureEach {
-        kotlinOptions {
-            jvmTarget = JavaVersion.VERSION_1_8
```

Comment:
Just remove `kotlinOptions` due to its deprecation warning, using the default `jvmTarget` (`JavaVersion.VERSION_1_8` level).